src/pages/login.vue

```diff
@@ -33,13 +53,13 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['configuration'])
+    ...mapGetters(['configuration']),
+    backgroundStyle () {
+      return `background-image: linear-gradient(${this.configuration.theme.colors.primary}, ${this.configuration.theme.colors.secondary});`
```

Comment:
return `background-image: linear-gradient(var(--v-primary-base), var(--v-secondary-base));` 

could be worth a try

Reply:
That's the snippet i was searching for... Thanks !
This one should work without computed property :)